Rules/CommonScripts/DefaultChatCommands.as

```diff
@@ -0,0 +1,62 @@
+#include "ChatCommandManager.as"
```

Comment:
Not reviewing the whole thing yet, the whole PR looks interesting. I'm wondering if this would be better implemented by a `.cfg` file that enumerates the scripts to load in some way (for which I don't see an obvious solution), as this would allow server owners to activate and deactivate some commands without requiring the server to be flagged as modded.

Reply:
The only way I can see it is possible for server owners to add new commands without making the server modded is by using `AddMod()`. Obviously, this is a horrible idea.

What I can do is give server owners the ability to disable commands (or enable commands that are disabled in vanilla) by modifying a `.cfg` file in their Cache folder. This config could even allow them to configure whether each command can be used during `sv_test` or only as an admin. More complex criteria would require using a mod.

Reply:
Yup, I didn't mean to suggest allowing to add custom commands. By definition that would and should be a mod.

> What I can do is give server owners the ability to disable commands (or enable commands that are disabled in vanilla) by modifying a .cfg file in their Cache folder. This config could even allow them to configure whether each command can be used during sv_test or only as an admin. If they want to add new commands or create more complex execution criteria, they would need to use a mod.

Sure, that would be fine imo.